### PR TITLE
Final report introduction

### PR DESCRIPTION
**Pull request by HarmenKroon**
As a developer I want to have an Introduction in the final report, so that a reader knows what the report is about and what we will be discussing in the final report.

This PR also includes a bibliography

### CODE REVIEW

**Comment by jannelie** [comment]
Approved. Looks great!

[Approved with PullApprove](https://pullapprove.com/nskartoredjo/TI2806/pull-request/61/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=nskartoredjo/TI2806)

**Comment by jannelie** [comment]
Approved

File: docs/final report/Introduction.tex [hunk 1]
```diff
@@ -1,4 +1,9 @@
 \section{Introduction}
 
+This project is build around the Tygron Engine; an urban planning 3D software product. It requires different stakeholders to work together in real life locations. This allows all parties to simulate how certain decisions, might influence everyone participating. All planning is done in real time, which makes the simulation more real, but creates problems requiring every stakeholder to be physically present during the simulation. This is commonly done through Charretes\cite{Todd13}. To combat this problem we are tasked with designing and creating a virtual human, which simulates a real stakeholder by the use of artificial intelligence programming. This virtual human is designed around a scenario created at the beginning of the project, but should be capable of acting in a multitude of scenarios in order to be of use to any actual simulation involving users of the Tygron Engine.\newline
```
Comment (nskartoredjo) [inline docs/final report/Introduction.tex]: `might influence everyone participating.` broken sentence.

`\newline` should be replace by a empty line

File: docs/final report/Introduction.tex [hunk 2]
```diff
@@ -1,4 +1,9 @@
...
 
-\newpage
\ No newline at end of file
+In this report we will go over a couple of aspects of the project, process and product.\newline
+First we will give an overview of the developed and implemented software product. Then a reflection is discussed on the product and process. This is done from a software engineering perspective. Thirdly, the developed functionalities are discussed. The following chapter contains a detailed explanation about our implementation of interaction design techniques. In the sixth section an evaluation of all the functional modules and the product in its entirety, including a failure analysis. Lastly an outlook on the project is given.
+
+
```
Comment (nskartoredjo) [inline docs/final report/Introduction.tex]: unnecessary empty lines

**Comment by nskartoredjo** [comment]
Approved

**Comment by ardwennem2** [comment]
Approved

**Pull request merged**